app/budget/BudgetCtrl.html

```diff
@@ -4,6 +4,1415 @@
 <ipa-header page-title="Support Staff Assignments"></ipa-header>
 
 <!-- handle lack of active teaching call -->
-<div>
-	I am a budget app
+<script type="text/javascript">
+        jQuery(document).ready(function($) {
+			 $('#cash-in-section').hide();
+            $('#cash-out').on('click', function(e) {
+                console.log('showing courses');
+                $('#cash-out-section').show();
+                $('#cash-in-section').hide();
+            });
+            $('#cash-in').click(function(e) {
+                console.log('showing instructors');
+                $('#cash-out-section').hide();
+                $('#cash-in-section').show();
+            });
+			$('#addNotes').click(function(){
+			var $this = $(this);
+			$this.toggleClass('add-notes');
+			if($this.hasClass('add-notes')){
+				$this.text('Submit');
+			} else {
+				$this.text('Add New Notes');
+			}
+       		$('.notes-area').toggle();
+    		});
+        });
+ </script>
+<div class="main-content">
```

Comment:
I would add a class here on the 'root' div called 'budget-view', and then namespace all your CSS rules into the budget view class